explainer.md

```diff
@@ -578,15 +582,44 @@ images would be common.
 ## Natively Supporting Animating Box Decoration CSS Properties
 
 An alternate approach to the setup described in
-[Animating Box Decoration CSS Properties](#live-animatable-properties) is to
-support this natively in the browser by introducing a new `content-element()`
+[Animating Box Decoration CSS Properties](#animating-box-decoration-css-properties)
+is to support this natively in the browser by introducing a new `content-element()`
 function. This function would behave similarly to the `element()` function
 except skipping the following properties when painting the element: box
 decorations and visual effects which generate a stacking context. The image
 would also be sized to the element's content-box (as opposed to the border-box
-used by the element() function). The motivation for supporting this natively
-would be to make these properties animatable instead of requiring developers to
-implement it themselves.
+used by the element() function).
+
+The motivation for supporting this natively would be to make these properties
+animatable by default instead of requiring developers to implement it themselves.
+This is a consideration for future iterations of the feature.
+
+## Clipping of Effects
+
+The decorated bounding box used by the element() function clips effects which paint
+outside the element's border-box (like box-shadow, drop-shadow) when generating an
+element's image. Developers can avoid this by using the approach outlined in
+[Animating Box Decoration CSS Properties](#animating-box-decoration-css-properties) to
```

Comment:
That doesn't solve the problem for things like text-shadow, eg https://static-misc-3.glitch.me/overflow-demo/